cardano-config/src/Cardano/Config/Types.hs

```diff
@@ -163,7 +162,7 @@ instance FromJSON NodeConfiguration where
   parseJSON = withObject "NodeConfiguration" $ \v -> do
                 nId <- v .:? "NodeId"
                 ptcl <- v .: "Protocol" .!= RealPBFT
-                genFile <- v .: "GenesisFile" .!= "configuration/genesis/genesis.json"
+                genFile <- v .: "GenesisFile" .!= "genesis/genesis.json"
```

Comment:
If this is our default, and we're changing it, how about just "genesis.json", so not requiring any subdir.